src/lib.rs

```diff
@@ -949,11 +987,45 @@ impl<
             high_qc: high_qc.clone(),
             api: c_api.clone(),
         };
-        let _ = da_member.run_view().await;
+        async_spawn(async move { da_member.run_view() });
         let _da_replica = {};
-        // TODO tie all the tasks together and do correct book keeping.
-        #[allow(deprecated)]
-        nll_todo()
+        // TODO: ADD DA replica task handle and push it to `task_handles`.
```

Comment:
Looks like `task_handles` only includes handles of the DA leader and the next leader. Besides the replica handle which we can't add now, I think we could add the handles of the consensus leader and DA member to `task_handles`.

Reply:
The consensus leader is together with the DA leader task since they are sequential.  So `leader_handle` includes both the DA leader and consensus leader.  I did forget the DA memeber